src/nettle-soup.md

```diff
@@ -0,0 +1,31 @@
+# Nettle Soup  
+
+Originally from a finnish book _Saiturin Keittiö_ (The Cheapskates Kitchen) from 1992.
+
+- 🍳 Cook time: 20 min
+- 💲  Cost <3€ (if nettles are self foraged)
+
+## Ingredients
+
+- 3 tbsp butter
+- 2l of fresh nettle sprouts
```

Comment:
```suggestion
- 2L of fresh nettle sprouts
```

Maybe it looks different when formatted, but in text it is hard to tell if this is the number `21` or `2 litres`, so I'd go with a capital `L` when using just litres